dataformat-xml-dom/src/test/java/org/camunda/spin/impl/xml/dom/format/DomXmlDataFormatWriterTest.java

```diff
@@ -0,0 +1,86 @@
+package org.camunda.spin.impl.xml.dom.format;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.StringWriter;
+import javax.xml.transform.Templates;
+import org.camunda.spin.impl.util.SpinIoUtil;
+import org.camunda.spin.xml.SpinXmlElement;
+import org.junit.Test;
+import org.w3c.dom.Element;
+
+/**
+ * Test xml transformation in DomXmlDataFormatWriter
+ *
+ * @author Lars Uffmann, 2020-10-16
+ * @author Joern Muehlencord
+ */
+public class DomXmlDataFormatWriterTest {
```

Comment:
A test case that reproduces the existing behavior would also be useful here, as we would be able to assert the behavior for both with pretty print and without it.